mmv1/products/compute/api.yaml

```diff
@@ -15073,6 +15075,18 @@ objects:
           resource: 'SslCertificate'
           imports: 'selfLink'
           description: 'The SSL certificates used by this TargetHttpsProxy'
+      - !ruby/object:Api::Type::String
+        name: 'certificateMaps'
+        description: |
+          A reference to the CertificateMap resource that identifies a certificate map 
+          associated with the given target proxy. This field can only be set for global target proxies.
```

Comment:
Do we want to add `If set, sslCertificates will be ignored.`?

Reply:
I changed `at_least_one_of` into `exactly_one_of` which should clarify the dependents of those two fields. However i could still add this information, what do u think?